utopia-remix/app/models/project.server.ts

```diff
@@ -23,12 +23,15 @@ export async function listProjects(params: { ownerId: string }): Promise<Project
   })
 }
 
-export async function getProjectOwnerById(params: { id: string }): Promise<string | null> {
+export async function getProjectOwnerById(
+  params: { id: string },
+  config: { allowDeleted: boolean } = { allowDeleted: false },
```

Comment:
the wording is a bit off, this is not "allowing: but rather _"including"_ the soft-deleted prpject

Reply:
fixed